Inpsyde/Sniffs/CodeQuality/HookPriorityLimitSniff.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the "php-coding-standards" package.
+ *
+ * Copyright (c) 2023 Inpsyde GmbH
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+declare(strict_types=1);
+
+namespace Inpsyde\Sniffs\CodeQuality;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class HookPriorityLimitSniff implements Sniff
+{
+    /**
+     * @return list<int>
+     */
+    public function register(): array
+    {
+        return [T_STRING];
+    }
+
+    /**
+     * @param File $phpcsFile
+     * @param int $stackPtr
+     * @return void
+     *
+     * phpcs:disable Inpsyde.CodeQuality.ArgumentTypeDeclaration
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        // phpcs:enable Inpsyde.CodeQuality.ArgumentTypeDeclaration
+
+        $tokens = $phpcsFile->getTokens();
+        $functionName = $tokens[$stackPtr]['content'] ?? '';
+
+        if ($functionName !== 'add_filter' && $functionName !== 'add_action') {
+            return;
+        }
+
+        $parameters = $this->stringParameters($phpcsFile, $stackPtr + 1);
+
+        if (in_array('PHP_INT_MAX', $parameters, true) && $functionName === 'add_filter') {
+            $phpcsFile->addWarning(
```

Comment:
Not sure about what do you expect here to being improved. I think I printed the message the same way we do in every other sniff we have. Am I missing something? 🤔

Reply:
MessageHelper was not used when I introduced PHPUtils.
The reason is that it is only helpful when you have replacements in the message, otherwise it requires a longer signature without real benefit, IMO.
Anyway, considering it is mnot used anywhere I think it is also fine not using it here.